src/wallet/api/wallet.cpp

```diff
@@ -1349,7 +1359,7 @@ void WalletImpl::doRefresh()
     try {
         // Syncing daemon and refreshing wallet simultaneously is very resource intensive.
         // Disable refresh if wallet is disconnected or daemon isn't synced.
-        if (daemonSynced()) {
+        if (daemonSynced() || m_wallet->light_wallet()) {
```

Comment:
Does this need to check for light_wallet() first to avoid trying to sync daemon on light wallets ?